ui/components/ui/box/box.test.js

```diff
@@ -9,8 +9,8 @@ import {
   ALIGN_ITEMS,
   JUSTIFY_CONTENT,
   TEXT_ALIGN,
-  SIZES,
```

Comment:
I'm just as amazed as you are the we only use `SIZES` for `bordeRadius` in the Box component 😮